src/components/ecl-indicator/ecl-indicator.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component, Prop, h, Element } from '@stencil/core';
+import getAssetPath from '../../utils/assetPath';
+declare const ECL: any;
+
+@Component({
+  tag: 'ecl-indicator',
+  styleUrls: {
+    ec: './build/styles/ecl-indicator-ec.css',
+    eu: './build/styles/ecl-indicator-eu.css',
+  },
+  shadow: false,
+  scoped: true,
+  assetsDirs: ['build'],
+})
+export class EclIndicator {
+  @Element() el: HTMLElement;
+  @Prop() value: string = '';
+  @Prop() eclScript: boolean = false;
+  @Prop() styleClass: string = '';
+
+  getClass(): string {
+    const styleClasses = ['ecl-indicator', this.styleClass];
+
+    return styleClasses.join(' ').trim();
+  }
+
+  componentDidRender() {
+    if (this.eclScript) {
+      const src = getAssetPath('./build/scripts/ecl-indicator.js');
+      const existing = document.querySelector(`script[src="${src}"]`);
+      if (existing) {
+        existing.remove();
+      }
+
+      const script = document.createElement('script');
+      script.src = src;
+      script.onload = () => {
+        const indicator = new ECL.Indicator(this.el);
```

Comment:
i'm only realizing this now:
this way we initialize the script on the custom element, instead we need it on its firstElementChild, in this case .ecl-indicator